src/v2/utils/utils.ts

```diff
@@ -31,6 +31,26 @@ import type {
   UnknownType,
 } from '../types/types';
 
+export const uiConfig = {
```

Comment:
So `globalUnreadCountLimit` is configurable through props already. Rest of the constants are quite fragile, you mess with  one a bit and it can mess up 2-way scroll experience. So want to keep these as non configurable for now. Only to move to props, if really really needed by someone.